Tools/ardupilotwaf/boards.py

```diff
@@ -524,6 +524,7 @@ def __init__(self):
         self.board_name = 'px4fmu-v1'
         self.px4io_name = 'px4io-v1'
         self.romfs_exclude(['oreoled.bin'])
+        self.with_uavcan = True
```

Comment:
I guess this is still here because you couldn't fix the errors with Travis. In previous comments I've pointed where code needs to be protected with the macro so that a build without CAN can be successful. If you still can't make it work, please ping me to look at the Travis error.